kubernetes/config/samples/etcd-bitnami.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
```

Comment:
Sorry, what is the difference between the two ETCD YAMLs?

Reply:
Oh. I should exclude this file. `quay.io/coreos/etcd:latest` in etcd.yaml was not abled to be fetched in one day. quay.io service return 500 and then I move to dockerhub. We should keep `etcd.yaml` that's the image from official etcd team